client/src/components/App/Home/MakeBooking.styles.ts

```diff
@@ -9,13 +9,21 @@ export default styled.div`
     padding: 3rem 3rem 4rem;
   }
 
-  > h2 {
-    margin: 0 2rem 0 0;
+  > .header-area {
+    display: flex;
+    flex-direction: inherit;
+    > h2 {
+      margin: 0 2rem 0 0;
+
+      color: ${(props) => props.theme.palette.primary.main};
+      font-size: 2.4rem;
+      line-height: 2.4rem;
+      font-weight: 400;
+    }
 
-    color: ${(props) => props.theme.palette.primary.main};
-    font-size: 2.4rem;
-    line-height: 2.4rem;
-    font-weight: 400;
+    > .change-office-link {
+      margin-top: 1rem;
```

Comment:
Just wondering if we should be using `align-items: baseline` here rather than using margin to get the vertical alignment? https://css-tricks.com/snippets/css/a-guide-to-flexbox/#prop-align-items

@chapmanio your thoughts?

Reply:
with align-items: baseline
![image](https://user-images.githubusercontent.com/18725139/88061169-0c7be800-cb5f-11ea-97e8-f6e84877b177.png)

vs 

original margin approach
![image](https://user-images.githubusercontent.com/18725139/88061270-31705b00-cb5f-11ea-9021-5fbdf750e295.png)

Reply:
Yeah, fine, stick with what you've got. I'd be interested why this doesn't work as I'm naïvely assuming @chapmanio

Reply:
will stick with what i have got and merge in now before the branch goes stale. Can retrospectively pickup with rich if required.